packages/browsers/.mocharc.cjs

```diff
@@ -3,4 +3,5 @@ module.exports = {
   spec: 'test/build/**/*.spec.js',
   exit: !!process.env.CI,
   reporter: 'spec',
+  timeout: '10000ms',
```

Comment:
Let's be consistent with timing we should use number + underscore (Maybe a EsLint Rule would be nice - quick search -> [Fixer](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/numeric-separators-style.md), let's discuss offline).
```suggestion
  timeout: 10_000,
```